base/uk.ac.stfc.isis.ibex.ui.dae/src/uk/ac/stfc/isis/ibex/ui/dae/spectraplots/PersistedLocalChannel.java

```diff
@@ -25,6 +31,8 @@
 	
 	private static final WritableFactory WRITE_FACTORY = new WritableFactory(OnInstrumentSwitch.NOTHING);
 	private static final ObservableFactory OBS_FACTORY = new ObservableFactory(OnInstrumentSwitch.NOTHING);
+	private static final ScheduledExecutorService EXECUTOR = Executors.newSingleThreadScheduledExecutor(
+            new ThreadFactoryBuilder().setNameFormat("write-to-PV-%d").build());
```

Comment:
```suggestion
            new ThreadFactoryBuilder().setNameFormat("write-to-css-local-PV-%d").build());
```

or maybe

```suggestion
            new ThreadFactoryBuilder().setNameFormat("PersistedLocalChannel-worker-%d").build());
```

Which are a bit more explicit than `write-to-pv`, which is a bit too generic really (we have a *lot* of threads in IBEX that "write to pvs" - that's basically all the client does!)